packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -434,6 +434,7 @@ async function startCompass(opts: StartCompassOptions = {}): Promise<Compass> {
   process.env.DEBUG = `${process.env.DEBUG ?? ''},mongodb-compass:main:logging`;
   process.env.MONGODB_COMPASS_TEST_LOG_DIR = path.join(LOG_PATH, 'app');
   process.env.CHROME_LOG_FILE = chromedriverLogPath;
+  process.env.SHOW_WELCOME = 'false';
```

Comment:
If we do this, can we pick a Compass-specific environment variable name?

Can we maybe test the actual flow here instead, where we just close the welcome modal if it appears at startup?

Reply:
We'd have to wait to see if it appears which means we have to timeout. In the past 5 or 10 seconds wasn't enough because in CI there are random pauses and generally no real consistency in how long it takes to run tests. Which meant every test where we open compass gained 10+ seconds of waiting which quickly added up and was really painful when working on.

Plan is to have just one test that tests the actual flow - in time to first query. Working on it.

Reply:
Another problem with it is that it changes test behaviour / timing based on the order of tests. We either have to clear everything always (and then how do we test things like favourites being stored?) and then always have the first run experience on every test and not be able to test the usual flow (important for things like time to first query) or you have things like the order matters so suddenly you focus a test or file and you get the welcome modal where you normally never do.

Being able to turn it on/off was the best I could come up with when we had the tour.